modules/installation-azure-day2-operations-diskencryptionsets.adoc

```diff
@@ -0,0 +1,83 @@
+//Module included in the following assemblies:
+//
+// * installing/installing_azure/enabling-disk-encryption-sets-azure.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="preparing-disk-encryption-sets-day2-operator_{context}"]
+= Preparing an Azure Disk Encryption Set for Day2 Operator
+
+The {product-title} installation program can use an existing Disk Encryption Set with a user-managed key. To enable this feature, create a `DiskEncryptionSet` in Azure and provide the key to the installation program. 
```

Comment:
Based on https://github.com/openshift/openshift-docs/pull/72130#discussion_r2053562072, suggest to update here too, something like:
If disk encryption is not configured during installation, you can enable disk encryption with platform-managed keys on each node when cluster is up and running.